solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/ZkStateReader.java

```diff
@@ -105,7 +104,6 @@ public class ZkStateReader implements SolrCloseable {
   public static final String MAX_WAIT_SECONDS_PROP = "maxWaitSeconds";
   public static final String STATE_TIMESTAMP_PROP = "stateTimestamp";
   public static final String COLLECTIONS_ZKNODE = "/collections";
-  public static final String LIVE_NODES_ZKNODE = "/live_nodes";
```

Comment:
It's backward incompatible to remove this public static variable. May be we can deprecate this and remove later

Reply:
Shrug; it's also rather internal to SolrCloud internals.

Reply:
I know, but it's in SolrJ. So, it could be used by others . It doesn't hurt us to just keep it around as `@Deprecated`